scss/footer.scss

```diff
@@ -19,3 +19,25 @@
   font-size: 12px;
   padding: 8px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .footer {
+    margin-top: 350px;
+    padding: 55px;
+  }
+
+  .footer-logo {
+    width: 500px;
```

Comment:
I think that this will break the whole layout.
Think about it, 
You have cell phones that have 300px as width, so the footer will be totally broken in that resolutions.